stackslib/src/chainstate/nakamoto/coordinator/mod.rs

```diff
@@ -842,10 +842,11 @@ impl<
                     self.get_nakamoto_reward_cycle_info(header.block_height - 2)?;
```

Comment:
bit of a nit: mind making header.block_height - 2 into a variable to be used below in the error message as well.

Should we also have the -2 converted to a staturating sub? I don't imagine we ever would have an underflow, but just to be save?